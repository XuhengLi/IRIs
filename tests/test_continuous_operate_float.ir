int main()
	float a,b,c
	1.23
	|
	a 
	|
	b
	|
	+2.12
	|
	c
	printf(a)
	printf(b)
	printf(c)
Siri
