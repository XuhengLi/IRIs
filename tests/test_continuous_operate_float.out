1.23
1.23
3.35
